scout/server/blueprints/cases/views.py

```diff
@@ -993,15 +993,15 @@ def add_case_group(institute_id, case_name):
 
     controllers.add_case_group(store, current_user, institute_id, case_name, group_id)
 
-    return redirect(request.referrer)
+    return redirect(request.referrer + "#case_groups")
```

Comment:
:+1: We should really do that for more elements further down on the page! I was pleasantly surprised when testing. 😸

Reply:
I agree, little by little!